discord/guild.py

```diff
@@ -1894,16 +1916,14 @@ async def edit(
             .. versionchanged:: 2.0
 
                 Now accepts an enum instead of :class:`str`.
-        rules_channel: Optional[:class:`TextChannel`]
+        rules_channel: :class:`TextChannel`
```

Comment:
Why are you removing the `Optional` aspect of this?

Reply:
It cannot be `None` if community is enabled, but I noticed it does when it's not a community, my fault I'll revert this and this for `public_updates_channel`.